src/main/java/codeu/controller/FileUploadServlet.java

```diff
@@ -0,0 +1,41 @@
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+
+public class FileUploadServlet extends HttpServlet {
+
+	@Override
+	public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+
+		//get the file chosen by the user
+		Part filePart = request.getPart("fileToUpload");
+		
+		//get the InputStream to store the file somewhere
+	    InputStream fileInputStream = filePart.getInputStream();
```

Comment:
We should check that the filePart is not null before we do this.

Reply:
I did: 
		if(request.getPart("photo") != null){
		Part filePart = request.getPart("photo");
		}
Do my brackets need to include the next line as well?